clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -132,6 +132,118 @@ goodbye-world"
     );
 }
 
+#[test]
+fn suggest_argument_value() {
+    let mut cmd = Command::new("dynamic")
+    .arg(
+        clap::Arg::new("input")
+            .long("input")
+            .short('i')
+            .value_hint(clap::ValueHint::FilePath),
+    )
+    .arg(
+        clap::Arg::new("format")
+            .long("format")
+            .short('F')
+            .value_parser(["json", "yaml", "toml"]),
+    )
+    .args_conflicts_with_subcommands(true);
+    
+    {
+        use std::fs;
+
+        let testdir = snapbox::dir::DirRoot::mutable_temp().unwrap();
+        let testdir_path = testdir.path().unwrap();
+
+		fs::write(testdir_path.join("a_file"), "").unwrap();
+		fs::write(testdir_path.join("b_file"), "").unwrap();
+		fs::create_dir(testdir_path.join("c_dir")).unwrap();
+		fs::create_dir(testdir_path.join("d_dir")).unwrap();
```

Comment:
Is the indentation off?

Reply:
That's weird!

Reply:
Looks like my tabs didn't get automatically converted to four spaces.